hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/TestCleaner.java

```diff
@@ -378,9 +374,8 @@ public void testCleanNonPartitionedTable() throws IOException {
     try (SparkRDDWriteClient client = new SparkRDDWriteClient(context, writeConfig)) {
       String instantTime;
       for (int idx = 0; idx < 3; ++idx) {
-        instantTime = client.createNewInstantTime();
+        instantTime = client.startCommit();
         List<HoodieRecord> records = dataGen.generateInserts(instantTime, 1);
-        WriteClientTestUtils.startCommitWithTime(client, instantTime);
```

Comment:
Will there be more cleanups on the usage of `WriteClientTestUtils.startCommitWithTime(client, instantTime)`?